apm-agent-plugins/apm-quartz/apm-quartz-common/src/test/java/co/elastic/apm/agent/quartzjob/AbstractJobTransactionNameInstrumentationTest.java

```diff
@@ -124,24 +107,6 @@ void testJobWithResult() throws SchedulerException {
         assertThat(transaction.getResult()).isEqualTo("this is the result");
     }
 
-    @Test
-    void testDirectoryScan() throws SchedulerException, IOException {
```

Comment:
[for reviewer] this was only relevant for quartz 2.x (hence the `Assumptions` call at start of test method.